src/widget/button.rs

```diff
@@ -22,17 +22,25 @@ use crate::{
 };
 
 use crate::piet::{FontBuilder, PietText, PietTextLayout, Text, TextLayout, TextLayoutBuilder};
+
+use crate::localization::LocalizedString;
 use crate::theme;
 use crate::{Point, RenderContext};
 
+/// The text for the label; either a localized or a specific string.
+pub enum LabelText<T> {
+    Localized(LocalizedString<T>),
+    Literal(String),
+}
+
 /// A label with static text.
```

Comment:
This description feels a little stale, it's not necessarily static. And I think the distinction between this an `DynLabel` is getting fuzzier, not sure we still need both.

Reply:
Yea, it's interesting. `DynLabel` is definitely more flexible, but how useful that flexibility is in practice is hard to say.